todocore/src/main/java/com/knighttodo/todocore/service/DayTodoService.java

```diff
@@ -88,5 +89,11 @@ public DayTodoVO updateIsReady(UUID dayId, UUID dayTodoId, boolean isReady) {
         dayTodoVO = dayTodoMapper.toTodoVO(dayTodoRepository.save(dayTodoMapper.toTodo(dayTodoVO)));
         return experienceService.calculateExperience(dayTodoVO);
     }
+
+    @Transactional
+    public List<DayTodoVO> findByDayIdAliveOrderNumber(UUID dayId) {
```

Comment:
they all should have an order number, you don't need this method. And for GET requests you should use @Transactional(readOnly=true)